third_party/subscriptions-project/config.js

```diff
@@ -120,7 +120,7 @@ class Doc {
 
   /**
    * The `Document` node or analog.
-   * @return {!Node}
+   * @return {!Document}
```

Comment:
Not sure if this is going to break PWA's

Reply:
The type annotation? How can I check

Reply:
@qidonna this was your change doc.js must match the one in AMP so changing the type here could be an issue.  The amp version is here https://github.com/ampproject/amphtml/blob/06123ab062b2be8d04bdb3e232c7c02a8cffe09b/extensions/amp-subscriptions/0.1/doc-impl.js#L48

Reply:
Reverted in https://github.com/subscriptions-project/swg-js/pull/1501. @cvolzke4 could you help cherrypick into the release?

Reply:
I've applied the revert to this PR.

Reply:
jpettitt@ PTAL